migrations/sqls/20190324132907-dbdev-up.sql

```diff
@@ -0,0 +1,182 @@
+  addedBy int NOT NULL /*Self Reference to Person ID*/,
+  modifiedBy int NOT NULL /*Self Reference to Person ID*/,
+  FOREIGN KEY (addedBy) REFERENCES person(personId),
+  FOREIGN KEY (modifiedBy) REFERENCES person(personId)
+);
+/*drop recreate command*/
+#DROP TABLE person;
+/*Crate person Contanct table-Contact M:1 Person*/
+CREATE TABLE IF NOT EXISTS contact (
+  personId int NOT NULL, /*Foreign key reference to person table */
+  contactId int AUTO_INCREMENT PRIMARY KEY,
+  primaryContact varchar(32),
+  primaryCarrier varchar(32),
+  textMessages int, /*Would client like to recieve text messages from CU*/
+  emergencyContact varchar(32),
+  email varchar(128),
+  dateAdded DATETIME DEFAULT CURRENT_TIMESTAMP,
+  addedBy int NOT NULL, /*Foreign key reference to person table */
+  FOREIGN KEY (personId) REFERENCES person(personId),
+  FOREIGN KEY (addedBy) REFERENCES person(personId)
+);
+#DROP TAbLE contact;
+/*Create Demographics Table M:1 Person*/
+CREATE TABLE IF NOT EXISTS demographics (
+  personId int NOT NULL, /*foreing key reference to person table*/
+  demographicId int AUTO_INCREMENT PRIMARY KEY,
+  originCountry varchar(64),
+  languageHome varchar(32), /* Primary language spoken at home */
+  englishProficiency varchar(32), /*English Proficiency of Person*/
+  dateUSArrival datetime, /*date of us arrival, day need not be exact and can be defaulted to 1 or 15*/
+  employed varchar(5), /*employed binry Y-U-N*/
+  weeklyAvgHoursWorked tinyint, /*Can be null if not employed*/
+  emplopymentSector varchar(128), /*Can be null if not employed*/
+  hhSize tinyint, 
+  dependents tinyint, /*Replace people under 18 in house hold question*/
+  maritalStatus varchar(128), /*Not applicable for children age < 17*/
+  hhIncome varchar(128), /*Predefined income brackets at front end, not applicable for children*/
+  ethnicity varchar(32),
+  dateAdded DATETIME DEFAULT CURRENT_TIMESTAMP,
+  addedBy int NOT NULL,
+  FOREIGN KEY (personId) REFERENCES person(personId),
+  FOREIGN KEY (addedBy) REFERENCES person(personId)
+);
+#DROP TABLE DEMOGRAPHICS;
+/*Create Address table M:1 with people*/
+CREATE TABLE IF NOT EXISTS address (
+  personId int NOT NULL, /*Foreign key reference to person table*/
+  addressId int AUTO_INCREMENT PRIMARY KEY,
+  address1 varchar(128),
+  address2 varchar(128),
+  duelingType varchar(32), /*Dueling type, home, appartment, duplex, townhome*/
+  owned varchar(5), /*Dueling ownership type, rent or owned(Y/N)*/
+  city varchar(64),
+  zip varchar(5),
+  state varchar(2),
+  dateAdded DATETIME DEFAULT CURRENT_TIMESTAMP,
+  addedBy int NOT NULL,
+  FOREIGN KEY (personId) REFERENCES person(personId),
+  FOREIGN KEY (addedBy) REFERENCES person(personId)
+);
+
+/*
+Create users tables: I figured that later on, this application can be scaled to mainain, volunteers, board members, and maybe even donors and staff (HR management) 
+*/
+CREATE TABLE IF NOT EXISTS users (
+    userId int AUTO_INCREMENT PRIMARY KEY,
```

Comment:
very much nit (not important): the indentation changes here from 2 spaces above this line to 4 spaces on this line and below. Usually it doesn't matter which one is used, as long as it is consistent.